docs/src/inscriptions/recursion.md

```diff
@@ -42,6 +42,7 @@ The recursive endpoints are:
 - `/r/children/<INSCRIPTION_ID>/<PAGE>`: the set of 100 child inscription ids on `<PAGE>`.
 - `/r/children/<INSCRIPTION_ID>/inscriptions`: details of the first 100 child inscriptions.
 - `/r/children/<INSCRIPTION_ID>/inscriptions/<PAGE>`: details of the set of 100 child inscriptions on `<PAGE>`.
+- `/r/delegator/<INSCRIPTION_ID>`: inscription content of the delegator.
```

Comment:
I'm not super happy about the name. The "delegator" is the whole inscription, whereas this just returns its content.  I'm having trouble coming up with a better name though.`r/original-content`, `r/own-content`, `r/undelegated`, and `r/undelegated-content` are all ideas.

Reply:
Yeah good point delegator would be the entire envelope which will include the delegate tag. 
We could maybe further define what the content is called after ` OP_PUSH 0` Maybe call it the `body` or `payload`

`/r/body/:id`
`/r/payload/:id`